source/views/sis/components/multi-line-cell.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+
+import React from 'react'
+import {Cell} from 'react-native-tableview-simple'
+import {StyleSheet, Text, View} from 'react-native'
+import * as c from '../../components/colors'
+
+type Props = {
+	title: string,
+	leftDetail: string,
+	rightDetail: string,
+}
+
+export class MultiLineDetailCell extends React.PureComponent<Props> {
+	render() {
+		const {title, rightDetail, leftDetail} = this.props
+		return (
+			<Cell
+				cellContentView={
+					<View style={detailStyles.cellContentView}>
```

Comment:
Please pull this `<View>` into a variable defined before `return`, so we can lose a level of indentation here.